account-service/src/main/java/com/service/accountservice/config/JwtAuthenticationFilter.java

```diff
@@ -62,7 +62,5 @@ protected void doFilterInternal(
                 SecurityContextHolder.getContext().setAuthentication(usernamePasswordAuthenticationToken);
             }
         }
-        filterChain.doFilter(request,response);
```

Comment:
Unintentional removal, please revert this change

Reply:
will add it on the next commit